pkg/api/volume/handler.go

```diff
@@ -95,32 +96,58 @@ func (h *ActionHandler) Do(ctx *harvesterServer.Ctx) (interface{}, error) {
 	}
 }
 
-func (h *ActionHandler) validateExportVolume(storageClassName, pvcNamespace, pvcName string) string {
-	if storageClassName == "" {
-		return "Parameter `storageClassName` is required"
+func (h *ActionHandler) checkPVCInUsed(pvcNamespace, pvcName string) error {
+	// find any pod use this PVC (same validation on CDI)
+	index := fmt.Sprintf("%s-%s", pvcNamespace, pvcName)
+	if pods, err := h.pods.GetByIndex(indexPodByPVC, index); err == nil && len(pods) > 0 {
+		podList := []string{}
+		for _, pod := range pods {
+			indexedPod := fmt.Sprintf("%s/%s", pod.Namespace, pod.Name)
```

Comment:
We should really start implementing a helper function for that (e.g. https://github.com/harvester/vm-import-controller/pull/69/files#diff-988f959e35b780a27c5aede93f5acf9118827cac3d43da1a5cd648dfd78991c9R25)

Reply:
nice suggestion! I will improve that later!